pyodide-build/setup.cfg

```diff
@@ -35,7 +35,7 @@ install_requires =
     requests
     types-requests
     typer
-    auditwheel-emscripten==0.0.8
+    auditwheel-emscripten==0.0.9
```

Comment:
Not critical but we could also pin to a range of versions, e.g.
```
auditwheel-emscripten>=0.0.9,<0.1.0
```
so that if there are any bugs, they can be fixed without waiting for a new pyodide-build release. Which would be the advantage of having this as a separate package.

Reply:
Yeah, sounds good to me.